src/applications/accredited-representative-portal/containers/POARequestIndividualSearchPage.jsx

```diff
@@ -10,36 +10,150 @@ import SsnField from 'platform/forms-system/src/js/web-component-fields/SsnField
+  }
+  if (poaStatus === 'accepted_with_pending') {
+    return 'If you accept the pending request from the claimant, you will replace the current representative with the new one.';
+  }
+  if (poaStatus === 'accepted_with_declined') {
     return (
-      <p data-testid="poa-requests-table-fetcher-no-poa-requests">
-        We did not find an individual “{searchData.first_name}{' '}
-        {searchData.last_name}
-        ”, “{searchData.dob}
-        ”, “{searchData.ssn}” who has submitted a power of attorney request in
-        the past 60 days to you or your organizations.
-      </p>
+      <span>
+        To change POA, have the claimant submit a new POA request online using{' '}
+        {poaFormLink()}.
+      </span>
+    );
+  }
+  if (poaStatus === 'declined') {
+    return (
+      <span>
+        To establish POA, have the claimant submit a new POA request online
+        using {poaFormLink()}.
+      </span>
+    );
+  }
+  return null;
+};
+
+const SearchResults = ({ claimant, searchData }) => {
+  if (!claimant) {
+    return (
+      <>
+        <p data-testid="poa-requests-table-fetcher-no-poa-requests">
+          No result found for{' '}
+          <strong>
+            “{searchData.first_name} {searchData.last_name}”
+          </strong>
+          {', '}
+          <strong>“{searchData.dob}”</strong>
+          {', '}
+          <strong>
+            “***-**-
+            {lastFour(searchData.ssn)}”
+          </strong>
+        </p>
+        <va-banner
+          data-label="Info banner"
+          headline="How to establish power of attorney"
+          type="info"
+          className="home__banner"
+          visible
+        >
+          <p>
+            This individual may exist in the system, but if they have not
+            designated you as their representative, you cannot view their
+            information. To establish POA, have the claimant submit a POA
+            request online using {poaFormLink()}.
+          </p>
+        </va-banner>
+      </>
     );
   }
 
   return (
-    <ul
-      data-testid="poa-requests-card"
-      className="poa-request__list"
-      sort-column={1}
-    >
-      {poaRequests.map((request, index) => {
-        return <POARequestSearchCard poaRequest={request} key={index} />;
-      })}
-    </ul>
+    <>
+      <p data-testid="poa-requests-table-fetcher-poa-requests">
+        Showing result for <strong>“{searchData.first_name}“</strong>
+        {', '}
+        <strong>“{searchData.last_name}“</strong>
+        {', '}
+        <strong>“{searchData.dob}”</strong>
+        {', '}
+        <strong>
+          “***-**-
+          {lastFour(searchData.ssn)}”
+        </strong>
+      </p>
+      <h2>
+        {claimant.lastName}, {claimant.firstName}
+      </h2>
+      {claimant.city}, {claimant.state} {claimant.postalCode}
+      <br />
+      <br />
+      {claimant.poaStatus === 'pending' || claimant.poaStatus === 'declined' ? (
```

Comment:
This seems awkward. Could we base this UI on whether or not there is a `poa` rather than read from this intermediate CTA logic?